microshift_running_apps/microshift-embed-apps-on-rhel-edge.adoc

```diff
@@ -0,0 +1,79 @@
+
+[id="microshift-add-app-RPMs-to-rpm-ostree-image"]
+== Add application RPMs to an rpm-ostree image
+If you have an application that includes APIs, container images, and configuration files for deployment such as manifests, you can build application RPMs. You can then add the RPMs to your {op-system-ostree} system image.
+
+The following is an outline of the procedures to embed applications or workloads in an fully self-contained operating system image:
+
+* Build your own RPM that includes your application manifest.
+* Add the RPM to the blueprint you used to install {product-title}.
+* Add the workload container images to the same blueprint.
+* Create a bootable ISO.
+
+[id="microshift-add-app-manifests-to-image"]
+== Add application manifests to an image for offline use
+If you have a simple application that includes a few files for deployment such as manifests, you can add those manifests directly to a {op-system-ostree} system image.
+
+See the "Create a custom file blueprint customization" section of the following {op-system-ostree} documentation for an example:
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images#image-customizations_composing-a-rhel-for-edge-image-using-image-builder-command-line[Create a custom file blueprint customization]
+
+include::modules/microshift-embed-app-rpms-tutorial.adoc[leveloffset=+1]
+
+include::modules/microshift-preparing-to-make-app-rpms.adoc[leveloffset=+2]
+
+include::modules/microshift-building-apps-rpms.adoc[leveloffset=+2]
+
+include::modules/microshift-adding-app-rpms-to-blueprint.adoc[leveloffset=+2]
+
+//additional resources for adding app rpms to blueprint
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images#doc-wrapper[Composing a RHEL for Edge image using the Image Builder CLI]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images#network_based_deployments_workflow[Network-based deployments workflow]
+
+include::modules/microshift-embed-images-offline-use.adoc[leveloffset=+2]
+
+//additional resources for embedding images for offline use
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Some of the links reference 4.13 here. Should we use "latest" instead in the URLs?